layers/+lang/prolog/README.org

```diff
@@ -0,0 +1,92 @@
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+- [[#key-bindings][Key bindings]]
+  - [[#leader][Leader]]
+    - [[#consulting][Consulting]]
+    - [[#compiling][Compiling]]
+    - [[#formatting][Formatting]]
+    - [[#inserting][Inserting]]
+    - [[#help][Help]]
+    - [[#evaluating][Evaluating]]
+
+* Description
+This layer adds support for Prolog using the bundled Prolog mode for Emacs. In addition it also adds ediprolog support for better interaction with SWI-Prolog. 
+
+** Features:
+  - Designed for SWI-Prolog as a default, but can be used with other Prologs that Prolog mode supports. 
+  - Interactive consulting and compiling.
+  - Auto-formatting.
+  - Apropos and help lookup.
+
+    
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =prolog= to the existing =dotspacemacs-configuration-layers= list in this
+file.
+
+Note that the layer currently uses the Prolog mode that is bundled with Emacs. 
+If you want to use the separately maintained [[https://bruda.ca/_media/emacs/prolog.el][Prolog Mode by Stefan D. Bruda]]:
+- Place =prolog.el= in the =local/prolog= directory relative the layer directory.
+- When defining the =prolog-packages= for the layer, change =(prolog :location built-in)=
+ to =(prolog :location local)=.
+
+* Key bindings
+  
+** Leader
+*** Consulting
+
+| Key Binding | Description       |
+|-------------+-------------------|
+| ~SPC m s b~ | Consult Buffer    |
+| ~SPC m s f~ | Consult File      |
+| ~SPC m s p~ | Consult Predicate |
+| ~SPC m s r~ | Consult Region    |
+
+*** Compiling
+
+| Key Binding | Description       |
+|-------------+-------------------|
+| ~SPC m c b~ | Compile Buffer    |
+| ~SPC m c f~ | Compile File      |
+| ~SPC m c p~ | Compile Predicate |
+| ~SPC m c r~ | Compile Region    |
+| ~SPC m c c~ | Compile File      |
+
+*** Formatting 
+
+| Key Binding | Description   |
+|-------------+---------------|
+| ~SPC m =~   | Indent Buffer |
+
+*** Inserting
+
+| Key Binding | Description                                        |
+|-------------+----------------------------------------------------|
+| ~SPC m i m~ | Insert a Modeline for Module Specification         |
+| ~SPC m i n~ | Insert Newline and the Name for the Current Clause |
+| ~SPC m i p~ | Insert Predicate Template for the Current Clause   |
+| ~SPC m i s~ | Insert Predicate Spec                              |
+
+*** Help
+
+| Key Binding | Description                           |
+|-------------+---------------------------------------|
+| ~SPC m h a~ | Prolog Apropos for a Given String     |
+| ~SPC m h p~ | Online Help for the Atom under Cursor |
+
+*** Evaluating 
+
+| Key Binding | Description                               |
+|-------------+-------------------------------------------|
+| ~SPC m e e~ | Context Sensitive SWI-Prolog Interaction. |
+
+# Use GitHub URLs if you wish to link a Spacemacs documentation file or its heading.
```

Comment:
This is a comment block and should not remain in your layer file.

Reply:
Just checked your code and the block is still in there. Could you please double check?

Reply:
My apologies, the block should be removed now.